blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -295,6 +84,21 @@ namespace blueprint
             engine.invoke("__BlueprintNative__.dispatchEvent", eventType, std::forward<T>(args)...);
         }
 
+        //==============================================================================
+        void registerViewType(const juce::String& typeId, ViewManager::ViewFactory f)
```

Comment:
@nick-thompson after this gets merged I think registerViewType could do with a doc comment that gives a clear example of usage.

Reply:
👍 totally agree!